.github/workflows/ci-build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

Comment:
Thank you! Can you update the document too? > https://slack.dev/node-slack-sdk/#requirements
>This package supports Node v12 LTS and higher.

Reply:
@seratch - Yes, I see there are some docs that reference even older versions. I _think_ I got them all!